### PR TITLE
Automatically Forces the shuttle if it takes too long to launch.

### PR DESCRIPTION
**Pull request by Arrow768**
Emergency shuttles will forcefully depart 5 minutes after docking with the station.
Crew-Transfer shuttles will forcefully depart 3 minutes after docking with the station.

### CODE REVIEW

**Comment by Geevies** [comment]
Niche change, but can you also change the UI to display "Delay" instead of "Abort" or whatever it says?

**Comment by KingOfThePing** [comment]
Please give me a definition of "taking too long".

**Comment by Sindorman** [comment]
@KingOfThePing From the Code it seems as in 2 minutes from moment it is supposed to launch.

@Arrow768 This could be a problem to #5771, But I will adjust it later in my PR

**Comment by OneOneThreeEight** [comment]
Not sure if I really support this, but if this the middle ground solution, you should slap another minute on each of the timers.

**Comment by Arrow768** [comment]
The times were setup after discussion with the mins.

The crew transfer shuttle can not be held, because it is pretty similar to a train.
(I.e. it needs to be at certain locations at certain times; so you cant just walk up to the train driver and say "hey, can you wait for 2 minutes longer")

The emergency shuttle can be held for 2 minutes (after the originally planned departure.
That means that until the shuttle departs there have been 15 minutes to get to the shuttle dock and onto the shuttle.
Which is more than sufficient.
If the shuttle dock is vented/unpowered/damaged, then it would probs be a good idea to ensure it is operational before calling the shuttle.
(But thats the responsibility of the heads of staff)

**Comment by OneOneThreeEight** [comment]
>If the shuttle dock is vented/unpowered/damaged, then it would probs be a good idea to ensure it is operational before calling the shuttle.

10 minutes is usually enough to mini-nuke departures.

**Comment by Arrow768** [comment]
> 10 minutes is usually enough to mini-nuke departures.
Being able to hold the shuttle for a minute longer won't change that. 

If there has been sufficient ic reason to attack departures then security should secure it to prevent that scenario. 

If there isn't it's a ooc issue. 

This PR does not change that.

File: html/changelogs/arrow768-shuttleforce.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "The emergency shuttle will now force its launch if it is delyed for too long."
```
Comment (ParadoxSpace) [inline html/changelogs/arrow768-shuttleforce.yml]: delayed

**Comment by Arrow768** [comment]
Outdated contributor review, so the bot doesn't update awaiting merge automatically.

**Pull request merged**